lib/storage_tables/attachable/one.rb

```diff
@@ -16,13 +16,32 @@ class One < ActiveStorage::Attached::One
       #   person.avatar.attach(avatar_blob) # ActiveStorage::Blob object
       def attach(attachable, filename:)
         record.public_send("#{name}=", attachable, filename)
+        blob.save! && upload(attachable)
         # :nocov:
         return if record.persisted? && !record.changed? && !record.save
         # :nocov:
 
         record.public_send(name.to_s)
       end
 
+      def upload(attachable)
```

Comment:
Het uploaden van een File tijdens een transaction is onveilig aan de Blob ook aangemaakt in dezelfde transaction. Bovendien houdt het de transaction veel te lang open. Ik zou er wel voor zijn om een exception te raisen als deze method aangeroepen wordt terwijl we in een transaction zitten. Misschien iets voor een losse PR